<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindWidget</class>
 <widget class="QWidget" name="FindWidget">
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labelFind">
     <property name="text">
      <string>Fin&amp;d:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboBoxFind</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxFind">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxCaseSensitive">
     <property name="text">
      <string>C&amp;ase sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pushButtonBackward">
     <property name="toolTip">
      <string>Search backward</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="pdfviewer.qrc">
       <normaloff>:/icons/go-up.png</normaloff>:/icons/go-up.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pushButtonForward">
     <property name="toolTip">
      <string>Search forward</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="pdfviewer.qrc">
       <normaloff>:/icons/go-down.png</normaloff>:/icons/go-down.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonFind">
     <property name="text">
      <string>&amp;Find</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pushButtonClose">
     <property name="toolTip">
      <string>Close search panel</string>
     </property>
     <property name="icon">
      <iconset resource="pdfviewer.qrc">
       <normaloff>:/icons/dialog-cancel.png</normaloff>:/icons/dialog-cancel.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxFind</tabstop>
  <tabstop>checkBoxCaseSensitive</tabstop>
  <tabstop>pushButtonBackward</tabstop>
  <tabstop>pushButtonForward</tabstop>
  <tabstop>pushButtonFind</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources>
  <include location="pdfviewer.qrc"/>
 </resources>
 <connections/>
</ui>
